### DISCUSSION TITLE
what is the late collisions? and what are the reasons that cause late collisions?

### DISCUSSION THREAD

**Discussion by mason5052** [Kimjunkuk/SoftwareQualityAssurance#11] [Q&A]
Greeting,

I'm wondering that what is the late collisions? and what are the reasons that cause late collisions?

I'd appreciate it if provide me any advice. 

Thanks!

**Comment by Kimjunkuk** [comment]
Station A, Station B 
Maybe A station and B station at the same time send signal then B will be getting a late signal.
But when the medium is free then B will send it again
[64byte ][CSMA/CD][Wifi][CSMA/CA]
